index.html

```diff
@@ -27,18 +27,18 @@
         <button type="button" class ="calc-btn" id = "num">7</button>
         <button type="button" class ="calc-btn" id = "num">8</button>
         <button type="button" class ="calc-btn" id = "num">9</button>
-        <button type="button" class ="operator" id = "/"><i class="fas fa-divide" alt="/" id = "/"></i></button>
+        <button type="button" class ="operator" id = "/"><i class="fas fa-divide" alt="/" id = "divide"></i></button>
```

Comment:
Good job! Yes, making the id an english-readable id is much better than naming it by the operator (`-`, `+`, etc)